src/components/Label/Label.tsx

```diff
@@ -175,6 +177,7 @@ export const Label = React.forwardRef(function Label(
                 title={title}
                 data-qa={qa}
             >
+                {loading && <div className={b('animation-container')} />}
```

Comment:
If disabled, loading should not be visible

Reply:
Thanks, fixed https://github.com/gravity-ui/uikit/pull/2297/commits/d0f76a9e3d274311b38e4707c0dcb7f7626c3597